swabbie-aws/src/main/kotlin/com/netflix/spinnaker/swabbie/aws/instances/AmazonInstance.kt

```diff
@@ -20,21 +20,22 @@ import com.fasterxml.jackson.annotation.JsonTypeName
 import com.netflix.spinnaker.swabbie.aws.model.AmazonResource
 import com.netflix.spinnaker.swabbie.model.AWS
 import com.netflix.spinnaker.swabbie.model.INSTANCE
+import java.time.Instant
 import java.time.LocalDateTime
 import java.time.ZoneId
-import java.util.Date
 
 @JsonTypeName("amazonInstance")
 data class AmazonInstance(
   val instanceId: String,
   val imageId: String,
   val tags: List<Map<String, String>>,
-  private val launchTime: Date,
+  private val launchTime: Long,
   override val resourceId: String = instanceId,
   override val resourceType: String = INSTANCE,
   override val cloudProvider: String = AWS,
   override val name: String = instanceId,
-  private val creationDate: String? = LocalDateTime.ofInstant(launchTime.toInstant(), ZoneId.systemDefault()).toString()
+  private val creationDate: String? =
```

Comment:
Does this need to be `private val` since it's just being passed to the super constructor?

Reply:
You can't do passthrough values with data classes apparently